### PR TITLE
Update Lisk SDK to v6.0.0-alpha.10

### LINKED ISSUE #1405: Update Lisk SDK to v6.0.0-alpha.10
### Description

Update Lisk SDK to v6.0.0-alpha.10

### Acceptance Criteria

- Update SDK to v6.0.0-alpha.10

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?
This PR resolves #1405 

### How was it solved?

- [x] Update Lisk SDK to v6.0.0-alpha.10 dependencies on all microservices

### How was it tested?
Locally against `Lisk Core v4.0.0-alpha.8`, running on Lisk SDK v6.0.0-alpha.10

### CODE REVIEW

File: services/newsfeed/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "bluebird": "^3.7.2",
     "html-entities": "^2.3.2",
     "html-to-text": "^8.0.0",
-    "knex": "^2.0.0",
+    "knex": "^2.3.0",
```
Comment (nagdahimanshu) [inline services/newsfeed/package.json:36]: I think we can remove `knex` as a dependency as we are not using it directly. The current newsfeed implementation is directly using MySQL from framework.

File: Makefile [hunk 2]
```diff
@@ -105,8 +105,8 @@ build-tests:
 	cd ./tests && docker build --tag=lisk/service_tests ./
 
 build-local:
-	npm ci
-	cd ./framework && npm ci
+	npm i --registry https://npm.lisk.com
+	cd ./framework && npm i --registry https://npm.lisk.com
```
Comment (priojeetpriyom) [inline Makefile:109]: Could using `npm install` create an issue for cases when our code becomes incompatible with new libraries?

File: services/newsfeed/package.json (hunk 1 above)
Comment (sameersubudhi) [reply services/newsfeed/package.json]: Addressed!

File: Makefile (hunk 2 above)
Comment (sameersubudhi) [reply Makefile:109]: This change is temporary to bypass current dependency installation issues. This won't make it to the production.

**Review by priojeetpriyom — APPROVED** [review]

**Review by nagdahimanshu — APPROVED** [review]

**Pull request merged**